### DISCUSSION TITLE
如何一次获取动态渲染的Form表单数据

### DISCUSSION THREAD

**Discussion by git997-small** [ant-design/ant-design#42500] [General]
### What problem does this feature solve?
dear teacher:
Form.Provider中的Form组件只能点击对应的button才能获取到对应Form中绑定的数据 如何在Form外定义一个Button 点击按钮获取多个Form中的数据
thank you :)
### What does the proposed API look like?

点击按钮获取多个Form中的数组

**Comment by BoyYangzai** [comment]
@git997-small 
https://codesandbox.io/s/form-instance-forked-70wkm4?file=/demo.tsx
通过 Form 组件绑定 useForm 出来的 `form实例` 来保存 form实例，然后通过 `form.getFieldsValue(true)` 拿到所有数据
![image](https://github.com/ant-design/ant-design/assets/94534613/531b92b6-5b4a-4715-ae7b-a34274833bf7)
![image](https://github.com/ant-design/ant-design/assets/94534613/ae1bedfa-da2a-422d-b269-44abbd6edcd8)
![image](https://github.com/ant-design/ant-design/assets/94534613/f75677e1-c0cf-43cb-8f9a-fdae085fe417)


写了个 demo
https://codesandbox.io/s/form-instance-forked-70wkm4?file=/demo.tsx:1140-1165

form instance 文档
https://ant.design/components/form-cn#forminstance
